### DISCUSSION TITLE
macOS problem: no navbar when previewing + render does nothing

### DISCUSSION THREAD

**Discussion by yairmau** [quarto-dev/quarto-cli#4728] [Q&A]
On my Mac, when I preview there is no navbar, just the content of the page. This happens for any website, even the simplest one created in a new project.
Furthermore, when I type 'quarto render' on the terminal, nothing happens.

I used another mac with older OS version, there everything works.
Do you happen to know if this is caused by the latest macos update, and if so, how to fix this?

Thanks!!

**Comment by mcanouil** [comment]
Could you provide more information?
What's the OS version?
What's the architecture? ARM?
With which browser do you preview the website?
Are you using an IDE to render/preview? Which IDE?
Do you have a reproducible example?


Currently, without the answers there is no way to help you as it seems the issue is tied to your specific setup.
